src/api/app/views/webui/project/_basic_info.html.haml

```diff
@@ -14,6 +14,6 @@
 
   - if policy(project).update?
     .pt-4
```

Comment:
Same for package which doesn't have any top padding.

```suggestion
    .pt-2
```

Reply:
I don't think it's a good idea because, when the description is collapsed, the `Edit` and the `Show more` links will be displayed more or less overlapped (occupying the same area).

![Show_home_scp_paquete_Open_Build_Service(1)](https://user-images.githubusercontent.com/2581944/111668236-d7bb5000-8815-11eb-9304-750cc8d2c066.png)

So I will keep it with `pt-4` for both project and packages.

Reply:
Good point :slightly_smiling_face:

At the time of my comment, the Edit link for package didn't have any padding. it's fixed now :+1: